charts/everest/README.md

```diff
@@ -12,51 +12,82 @@ but if you do so, please be aware of the risks.
+* By default, all database operators are installed in your database namespace. You may override this by specifying one or more of the following: [`dbNamespace.pxc=false`, `dbNamespace.pg=false`, `dbNamespace.psmdb=false`].
+* We currently do not support installation without the use of chart hooks. I.e, the use of `--no-hooks` is not supported during installation.
 
+### 3. Retrieve the admin password
+
+Once the installation is complete, you may retrieve the admin credentials using the following command:
 ```sh
 kubectl get secret everest-accounts -n everest-system -o jsonpath='{.data.users\.yaml}' | base64 --decode  | yq '.admin.passwordHash'
 ```
 
-### Deploy your database namespace components
+You may open the Everest UI by port-forwarding the service to your local machine:
+
+```sh
+kubectl port-forward svc/everest -n everest-system 8080:8080
+```
+
+Notes:
+* The default username to login to the Everest UI is `admin`.
+* You may specify a different default admin password using `server.initialAdminPassword` parameter during installation.
+* The default admin password is stored in plain text. It is highly recommended to update the password using `everestctl` to ensure that the passwords are hashed.
+
+### 4. Deploy additional database namespaces
 
-Once Everest is running, we need to create a namespace for your databases and provision the necessary operators.
+After Everest is successfully running, you may create additional database namespaces using the `everest-db-namespace` Helm chart.
+If you had set `dbNamespaces.enabled=false` in the previous step, you may deploy a database namespaces using the following command:
 
 ```sh
-cat <<EOF | kubectl create -f -
-apiVersion: v1
-kind: Namespace
-metadata:
-  name: everest
-  labels:
-    app.kubernetes.io/managed-by: everest
-EOF
-helm install everest percona/everest-db-namespace --namespace everest
+helm install everest \
+    percona/everest-db-namespace \
+    --namespace everest
 ```
 
-### Uninstalling
+Notes:
+* By default, all database operators are installed in your database namespace. You may override this by specifying one or more of the following: [`dbNamespace.pxc=false`, `dbNamespace.pg=false`, `dbNamespace.psmdb=false`].
+* We currently do not support installation without the use of chart hooks. I.e, the use of `--no-hooks` is not supported during installation.
+
+### 5. Uninstalling
+
+If you created any additional database namespaces, you must delete them first before uninstalling Everest.
 
-As a first step, you must always clean up your database namespace(s) first, otherwise the deletion could get stuck.
 ```sh
-helm uninstall everest -n everest
-kubectl delete ns everest
+helm uninstall everest -n <your_db_namespace>
+kubectl delete ns <your_db_namespace>
 ```
 
-Then you can uninstall Everest itself:
+Then run the below command to uninstall Everest from your cluster.
 ```sh
 helm uninstall everest-core -n everest-system
 kubectl delete ns everest-system
 ```
```

Comment:
Maybe we could separate the 2 actions in 2 subsections (5.1 and 5.2) to make it more visible that there are 2 distinct steps depending on the installation.

Reply:
Yes, you're right. Fixed it :)